GOVERNANCE.md

```diff
@@ -3,39 +3,78 @@ status: draft
 ---
 # Commonhaus Foundation Project Governance
 
-The Commonhaus Foundation  (CF) is a collaborative space where contributors come together.
-While working in groups can present challenges, this document lays out the guidelines that foster unity and progress.
-
-[coc-reports]: CODE_OF_CONDUCT.md#handling-reports-and-escalations
-[coi-policy]: governance/conflict-of-interest.md
+Welcome to the governance documentation for the Commonhaus Foundation (CF). 
+
+This document is **Section 2** of [CF Bylaws][bylaws], outlines the structures, roles, and processes that guide our collaborative and open-source community. It's designed to ensure clarity, fairness, and inclusivity in our decision-making and community interactions.
+
+- [2.1 CF Membership and Roles](#21-cf-membership-and-roles)
+  - [2.1.1 Project Membership](#211-project-membership)
+  - [2.1.2 Project Leaders and Code Owners](#212-project-leaders-and-code-owners)
+  - [2.1.3 General Members](#213-general-members)
+  - [2.1.4 Membership Termination and Resignation](#214-membership-termination-and-resignation)
+- [2.2 CF Council](#22-cf-council)
+  - [2.2.1 Composition and Membership](#221-composition-and-membership)
+  - [2.2.2 Tenure and Transition](#222-tenure-and-transition)
+  - [2.2.3 Apportioning Duties Among Council Members](#223-apportioning-duties-among-council-members)
+- [2.3. CF Advisory Board](#23-cf-advisory-board)
+- [2.4 Decision Making and Voting](#24-decision-making-and-voting)
+  - [2.4.1 General Decision Making](#241-general-decision-making)
+  - [2.4.2 Elections](#242-elections)
+- [2.5 Record Keeping](#25-record-keeping)
+- [2.6 Code of Conduct](#26-code-of-conduct)
+- [2.7 Trademark Policy](#27-trademark-policy)
+- [2.8 Contributing](#28-contributing)
+
+[bylaws]: ./bylaws/README.md
+[coc]: ./CODE_OF_CONDUCT.md
+[coc-reports]: ./CODE_OF_CONDUCT.md#handling-reports-and-escalations
+[coi-policy]: ./bylaws/conflict-of-interest.md
+[Trademark Policy]: ./bylaws/trademark-policy.md
 [MEMBERS.yaml]: https://github.com/commonhaus/foundation-draft/blob/main/MEMBERS.yaml
+[ip-policy]: ./bylaws/ip-policy.md
+[records]: ./bylaws/legal-compliance.md#record-keeping
+[contrib]: ./CONTRIBUTING.md
+
+## 2.1 CF Membership and Roles
 
-## Table of Contents
+### 2.1.1 Project Membership
 
-- [Project Roles](#project-roles)
-- [Commonhaus Foundation Council](#commonhaus-foundation-council)
-- [Decision Making and Voting](#decision-making-and-voting)
-- [Code of Conduct](#code-of-conduct)
-- [Trademark Policy](#trademark-policy)
-- [Contributing](#contributing)
+We evaluate projects based on ownership, activity, usage, and originality. To be considered, projects must:
 
-## Project Roles
+- Be novel or a substantial fork of an existing project.
+- Show recent activity and/or significant usage as measured by downloads from a central package repository (Maven Central, GitHub Packages, etc.).
```

Comment:
Not necessarily wrong, but these phrases are quite vague - "substantial", "recent", "significant".